samples/bluetooth/mesh/light/child_image/hci_rpmsg/boards/thingy53_nrf5340_cpunet.conf

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Using a custom release configuration until debug functionalities
+# will be disabled in hci_rpmsg sample.
+
+CONFIG_RPMSG_SERVICE=y
+CONFIG_RPMSG_SERVICE_MODE_REMOTE=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+
+# Bluetooth configuration
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_LL_SW_SPLIT=y
+CONFIG_BT_COMPANY_ID=0x0059
+
+# Disable unused Bluetooth features
```

Comment:
Please set: `CONFIG_BT_BUF_CMD_TX_COUNT=4`
Lack of TX buffers could lead to errors on network core in release configurations. This is already updated in Zephyr.

Reply:
> This is already updated in Zephyr.

Is this set in the kconfig source files, so we can omit it from the conf files after the upmerge?

Reply:
The `CONFIG_BT_BUF_CMD_TX_COUNT=4` is already added to default configuration of Bluetooth hci_rpmsg sample in origin Zephyr.

Reply:
I think it could be a good idea to separate configuration overlays of hci_rpmsg to ensure that network core image configuration is properly updated when building sample for nRF53 DK or Thingy:53 (I see that we use part of the Bluetooth controller configuration that is not a default one - e.g. we use split LL).
That could also prevent build warnings related to Kconfig. As I noticed, nRF53 is already added to `sample.yaml`.

Reply:
Or maybe we could switch to using some default values to simplify the configuration?
@trond-snekvik what's your opinion on this one?

Reply:
+1 for default values in kconfig. The tricky part for us is that the network core kconfig context has no idea about what goes on in the parent image for the application core. Some of these options could be the default for all mesh-serving network core images, but we can't use the normal `default XYZ if BT_MESH` approach in kconfig. 

Perhaps we could propagate a conf file fragment to the network core through cmake?

Reply:
Maybe we could provide the missing configuration of network core as a Kconfig fragment for hci_rpmsg child image? It's already possible with multi-image build in NCS. See overlay (I) in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html for details

Reply:
I think there is no dependency mechanism between multiple images in NCS, so it could be difficult to do it in more generic way